envi/archs/i386/disasm.py

```diff
@@ -457,6 +457,11 @@ def getOperAddr(self, op, emu=None):
         if self.index != None:
             ret += (emu.getRegister(self.index) * self.scale)
 
+        if emu.imem_psize == 4:
```

Comment:
this might be better using something like:

ret &= e_bits.u_maxes[emu.imem_psize]
# e_bits is envi.bits, in case it isn't already imported in this module.
it is one line, it avoids potentially two if's and uses a list-lookup instead, and covers all sizes (in case Intel ever makes a 128-bit system based on x86)